packages/oui-navbar/src/brand/navbar-brand.component.js

```diff
@@ -7,13 +7,16 @@ export default {
         iconAlt: "@?",
         iconClass: "@?",
         iconSrc: "@?",
-        href: "@?"
+        href: "@?",
+        onClick: "<?"
```

Comment:
For callback function, prefer using this.

```suggestion
        onClick: "&"
```

We don't put `&?` because the fallback is already done with AngularJS with `angular.noop()`. Or else we need to check before calling it :)